src/providers/postgres/qgspostgresconn.cpp

```diff
@@ -107,7 +107,7 @@ QString QgsPostgresResult::PQfname( int col )
   return QString::fromUtf8( ::PQfname( mRes, col ) );
 }
 
-int QgsPostgresResult::PQftable( int col )
+Oid QgsPostgresResult::PQftable( int col )
```

Comment:
in some part of the master PR you used unsigned int instead of Oid... what's the reason?

Reply:
now its' clear abuut https://github.com/qgis/QGIS/pull/30192